src/GrassPokemon.java

```diff
@@ -0,0 +1,42 @@
+public class GrassPokemon extends Pokemon{
+
+    private double height;
+    private int weight;
+    public GrassPokemon(String name, String food, double height, int weight) {
+        super(name, food);
```

Comment:
Dit heb je heel goed toegevoegd. De super moet altijd als eerste in de constructor